bytecode/concrete.py

```diff
@@ -498,6 +498,8 @@ def compute_jumps(self):
 
         return modified
 
+    max_compute_jumps_steps = 10
```

Comment:
Could you please:
- make this variable private ?
- define it right after the class definition before the first method ?
- add a comment describing its purpose with a link to the relevant part of the CPython source ?

Reply:
done